src/main/cpp/Robot.cpp

```diff
@@ -82,6 +99,32 @@ void Robot::ExpectAutonomousEndConds() {
     }
 }
 
+void Robot::ClimbingSequenceSM() {
+    switch (m_state) {
+        case ClimbingStates::kSecondRung: {
+            solenoidTimer.Start();
```

Comment:
Should consider resetting this every time you enter a new state. Otherwise, it will carry over the existing time and it may not be 1 second. The timer should also be stopped once the state is exited.